### ISSUE TITLE
Multiple entries: Display open entry reports on entry pages

### ISSUE THREAD

**Issue by Shiroizu** [VocaDB/vocadb#1803]
The [entry report list](https://vocadb.net/Admin/ViewEntryReports?status=Open) is not visible to regular users.

Relevant entry reports could be displayed on entry pages:

![image](https://github.com/user-attachments/assets/84f4eab9-db24-40a8-8ac8-93b3fce5150e)

Entry report listing on entry pages would help:
- Moderators who'd quickly see if the relevant report has been closed.
- The editor who introduced the edit issue. The entry report notification can be easily missed. 
- Everyone on notifying that the entry has some issues.
- Everyone on whether the visible edit issue has been reported already.

Entry reports can contain sensitive/malicious information --> details should be invisible to regular users.

### Checklist (required)

Fill out the checklist.

- [x] Searched **[VocaDB/vocadb](https://github.com/VocaDB/vocadb/issues)** for duplicate issues.

**Comment by mn7216** [comment]
Adding extra context:
Users often neglect entry reports and continue making the same mistakes while editing. This ends up with a lot of extra work for those who have to clean up these mistakes in a majority of cases and sometimes might even result in said user being limited in extreme cases, which isn't ideal. The current "solution" is mainly to comment on the users page, since they are likely to retraffic it, but this is also sometimes ignored and adds unnecessary repetition

tl;dr this has the benefits of:

1. Reducing unnecessary work for anyone who deals with entry reports
2. Gives the user more opportunity to notice and correct their mistake, which is a large part of the reason behind entry reports.
3. Not creating unnecessary clutter and repetition as alternative methods like commenting on their profile.
4. Being (hopefully) more successful than the current methods of sending a notification to the inbox and (optionally) commenting on their profile.




As an addition or alternative to the above, perhaps it could be added next to the entry completion state notifications (ex: "This entry contains all the required information.") in red. This would be visible easily to the user while they are editing and would fit in with the UI.

**Comment by FinnRG** [comment]
I've noted the interest in this issue. I'll put it on the [roadmap for November](https://github.com/orgs/VocaDB/projects/4/views/1) together with other entry report system improvements.

**Comment by goomie** [comment]
Would it be useful for Trusted+ to have an option to make their reports public, in the event that they're requesting help for a complex task instead of notifying individual users about a mistake? (e.g. "this album should be split due to XYZ".)
Or should we instead encourage people to request help in the comments?
